packages/fax/src/rest/v3/emails/emails-api.ts

```diff
@@ -126,6 +131,16 @@ export class EmailsApi extends FaxDomainApi {
     return listPromise as ApiListPromise<Email>;
   }
 
+  /**
+   * List emails for a number
+   * List any emails for a number.
+   * @param { ListEmailsForNumberRequestData } data - The data to provide to the API call.
+   * @return {ApiListPromise<string>} - The list of emails for a given number
+   */
+  public listByNumber(data: ListEmailsForNumberRequestData): ApiListPromise<string> {
```

Comment:
Indeed, it has been added to the "Emails" tag since I released the Fax SDK. This is the same operation as the one in the "Services".
The page is updated now.